doc-site/docs/tutorials/hello-world.md

```diff
@@ -0,0 +1,137 @@
+# Hello World with Paladin
+
+This tutorial walks you through deploying and interacting with a simple `HelloWorld` smart contract using the Paladin SDK. The example demonstrates how to:
+1. Deploy the contract,
+2. Interact with it by calling its `sayHello` function,
+3. Retrieve and verify the emitted event.
+
+---
+
+## Running the Example
+
+The example code can be found in the [Paladin example repository](https://github.com/LF-Decentralized-Trust-labs/paladin/blob/main/example/public-storage).
```

Comment:
Is this the correct link?

Reply:
Yes, once this is merged.

Reply:
I'm still not sure this is right. It's pointing at the "public-storage" example, but shouldn't it point at "helloworld"?

Reply:
💀